Source/ThirdParty/GltfPipeline/processModelMaterialsCommon.js

```diff
@@ -226,6 +226,20 @@ define([
         return result;
     }
 
+    function modelHasVertexColors(gltf) {
+        var hasVertexColors = false;
+        ForEach.mesh(gltf, function(mesh) {
+            ForEach.meshPrimitive(mesh, function(primitive) {
+                ForEach.meshPrimitiveAttribute(primitive, function(attribute, semantic) {
+                    if (semantic.indexOf('COLOR') === 0) {
+                        hasVertexColors = true;
```

Comment:
Individual materials are allowed to have, or not have, vertex colors.  This branch appears to apply `hasVertexColors` globally to a model, so if one material is using them, the others go dark (by multiplying by zero apparently, in my testing).

Haven't tried this, but could you make the default color white, so if `hasVertexColors` is true but a particular material doesn't have `COLOR_0`, it acts as if it had white vertices instead of black?